bike-erp/backend/src/main/services/orderService/BikeOrderService.ts

```diff
@@ -31,60 +32,53 @@ export class BikeOrderService {
   public static addBike = async (bikeOrderList: Array<any>, email: string) => {
     return new Promise(async (resolve, rejects) => {
       let bikeIdList: number[] = [];
-
-      for (let i = 0; i < bikeOrderList.length; i++) {
-        // Verifying the data meets the requirements for the price
-        if (isNaN(bikeOrderList[i].price) || bikeOrderList[i].price < 0) {
+      bikeOrderList.forEach(async (bike) => {
```

Comment:
why was this reverted to the for each loop? I added the for loop to be able to apply this condition
``` if (i === bikeOrderList.length - 1) {
            await AccountingService.createAccountReceivable(
              bikeOrderList,
              bikeIdList,
              email
            );
          }```

Reply:
sorry I think it was a merge issue. will fix now

Reply:
@nasaku898 should be up to date now